### DISCUSSION TITLE
Wazuh rules don't work

### DISCUSSION THREAD

**Discussion by rambant777** [wazuh/wazuh#26878] [Q&A] [answered]
Hello!
Wazuh-manager gives no errors, the rules are visible in the web-interface, but judging by the logs the rules do not work, what should be changed for the rules to work correctly?

screen: 
![waz](https://github.com/user-attachments/assets/7379a720-30ed-4a60-90ea-53510aee25e6)

**Comment by Hasitha9796** [comment, marked as answer]
Hi @rambant777

I suspect your first rule isn’t working because it lacks an `<if_sid>` reference, potentially causing it to conflict with a default authentication failure rule. Ensure the first rule includes the if_sid tag to reference the specific rule ID for authentication failure.

For the second rule, remove the `<if_group>` tag, as you’ve already specified `<if_matched_id>` for authentication failure, making the group tag redundant.

Refer to the Wazuh documentation on creating custom rules for clarity: 
[Rules](https://documentation.wazuh.com/current/user-manual/ruleset/rules/index.html)
[Rules Syntax](https://documentation.wazuh.com/current/user-manual/ruleset/ruleset-xml-syntax/rules.html)
[Regular Expression Syntax](https://documentation.wazuh.com/current/user-manual/ruleset/ruleset-xml-syntax/regex.html)

Let me know if you need further assitance on this.
